internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -1604,6 +1611,31 @@ func ResetSystemDisk(seq runtime.Sequence, data interface{}) (runtime.TaskExecut
 	}, "resetSystemDisk"
 }
 
+// ResetUserDisks represents the task to reset the user disks.
+func ResetUserDisks(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFunc, string) {
+	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) (err error) {
+		for _, disk := range r.Config().Machine().Disks() {
```

Comment:
this looks like not what we were discussing - we should be resetting all disks but Talos system disk, like Sidero Agent does.

users might be running Rook/Ceph, and `Config().Machine().Disks()` will be empty